pkg/util/strings_test.go

```diff
@@ -1,37 +0,0 @@
-package util
-
-import (
-	"reflect"
-	"testing"
-)
-
-func TestUniqueStrings(t *testing.T) {
```

Comment:
@miminar because it is not used anywhere and I was refactoring the string util here.